### PR TITLE
Another try of a smoother intro what Chisel is

### PR DESCRIPTION
**Pull request by schoeberl**
Incorporated suggestions from @edwardcwang

### CODE REVIEW

**Comment by schoeberl** [comment]
As I have no write access to this repo, someone else needs to do the merge.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Chisel3
-Chisel3 is a new Firrtl based chisel.
-It is currently in BETA VERSION, so some Chisel features may change in the coming months.
+
+Chisel is a new hardware construction language to support adavanced hardware design and circuit generation.
```
Comment (edwardcwang) [inline README.md]: Typo: advanced

Comment (ccelio) [inline README.md]: adavanced > advanced

Comment (edwardcwang) [reply README.md]: Duplicate "code" review

**Comment by schoeberl** [comment]
Ok, how complicated does it get to improve the README? Don't know the design flow well enough. Do I need to do another pull request just to fix this typo? Or can this be done in the current pull request?

If this minor improvment gets to complicated to be merged in, I will simply stop contributing pull requests to the project.

Cheers,
Martin

**Comment by ccelio** [comment]
Just make another commit to your branch that you are pulling from and it'll update the PR automatically.

**Comment by ucbjrl** [comment]
You can

Add more commits by pushing to the patch-1
<https://github.com/schoeberl/chisel3/tree/patch-1> branch on
schoeberl/chisel3 <https://github.com/schoeberl/chisel3>

**Comment by schoeberl** [comment]
Thanks @ccelio and @ucbjrl, this was an easy exercise :-)

**Comment by ccelio** [comment]
Awesome, glad to see updating the PR worked. Thanks for improving our documentation!

**Comment by schoeberl** [comment]
review and merge now ;-)

**Pull request merged**